src/components/VaccinationEligibilityBlock/VaccinationEligibilityBlock.tsx

```diff
@@ -1,34 +1,43 @@
-import React from 'react';
-import { Heading2, Paragraph } from 'components/Markdown';
-import TabsPanel, { TabInfo } from 'components/TabsPanel';
+import React, { useState } from 'react';
+import { HashLink } from 'react-router-hash-link';
 import { Region, getStateName } from 'common/regions';
+import { COLOR_MAP } from 'common/colors';
+import { assert } from 'common/utils';
 import ExternalLink from 'components/ExternalLink';
+import { Heading2, Paragraph } from 'components/Markdown';
+import TabsPanel, { TabInfo } from 'components/TabsPanel';
 import { trackEvent, EventCategory, EventAction } from 'components/Analytics';
-import { getEligibilityInfo } from './utils';
 import { getVaccinationDataByRegion } from 'cms-content/vaccines';
-import ButtonBlock from './ButtonBlock';
+import { getEligibilityInfo, getRegionState } from './utils';
 import {
   Container,
   Section,
   Source,
 } from './VaccinationEligibilityBlock.style';
 import EligibilityPanel from './EligibilityPanel';
-import { COLOR_MAP } from 'common/colors';
+import OpenInNewIcon from '@material-ui/icons/OpenInNew';
+import { StyledLinkButton, ButtonsContainer } from './ButtonBlock.style';
+import { EmailAlertIcon } from 'components/EmailAlertsFooter/EmailAlertsFooter.style';
+import { scrollWithOffset } from 'components/TableOfContents';
 
 const VaccinationEligibilityBlock: React.FC<{ region: Region }> = ({
   region,
 }) => {
+  const state = getRegionState(region);
+  assert(
+    state,
+    `VaccinationEligibilityBlock cannot be used for locations with more than one state`,
+  );
+
   // TODO: Assert that the region is a county, state or a single-state metro
-  const stateName = getStateName(region);
-  const eligibilityData = getEligibilityInfo(region);
+  const stateName = getStateName(state);
```

Comment:
Can you do `state.fullname` and then `stateName` won't be `null`-able and you can drop the `Your state` fallbacks below?